OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
Note: We don't have an existing maintainer file here unless I missed something, so to be clear this list is just copied from `etcd-io/etcd`.   Please let me know if any alterations should be made.